kubernetes/samples/charts/traefik/README.md

```diff
@@ -91,6 +91,80 @@ TLS_PORT=`kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.po
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:${TLS_PORT}/testwebapp/
 ```
 
+## SSL Termination at Ingress Controller
+This sample demonstrates how to terminate SSL trafik at ingress contoller to access WebLogic console thru SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on WebLogic Domain level
+
+If you are using wdt to configure WebLogic Domain, you need to add the following resource section at the domain level to model yaml file.
+```
+resources:
+     WebAppContainer:
+         WeblogicPluginEnabled: true
+```
+If you are using WLST script to configure domain, following modification needed to respective py script.
```

Comment:
If you are using WLST script to configure domain, following modification needed to respective py script.-> If you are using a WLST script to configure the domain, then the following modifications are needed to the respective PY script.

Reply:
done